### ISSUE TITLE
Service people need their own accesses

### ISSUE THREAD

**Issue by Everturning** [space-wizards/space-station-14#26365]
## Description
Zookeeper should get zookeeper access to make their zoos more secure and unable to be loosed by the chef or janitor, the janitor shouldn't be able to waltz into botany and fuck up their setup (as they can on Cluster station), and so on

**Reproduction**
>be zookeeper on oasis
>your average service worker walks in
>looses all animals cause they're curious

**Screenshots**

**Additional context**
I know that currently zookeeper isnt an available in main maps but it still was annoying when Oasis was set up on mander the other night

**Comment by lzk228** [comment]
if @Emisse approve, ill fo it

**Comment by PJB3005** [comment]
Zookeeper is a station-specific role so that needs a way to hide the access when not using that role, the issue with Cluster is a mapping issue.

**Comment by PJB3005** [comment]
Moving the Cluster issue to https://github.com/space-wizards/space-station-14/issues/26396

**Comment by IProduceWidgets** [comment]
I agree it needs its own access, with service access, the most restrictive one the zookeeper currently gets, too many people can accidentally (or intentionally) release the animals ruining zookeeper fun.

**Comment by Everturning** [comment]
any word on if this can be completed?

**Comment by Hitlinemoss** [comment]
#41741 removed Zookeeper so this issue is technically closed now (though someone else will need to mark it as closed since I don't actually have the ability to do so myself).

If we ever do re-add Zookeeper as a formal role in the future, though, we _should_ make sure that it does, in fact, have its own unique access type. Especially since it's much easier to just map Zookeeper-access airlocks in the first place than to make them Service access or something at first and need to go back and fix them later.
